processors/src/EUTelAlignGBL.cc

```diff
@@ -152,39 +152,31 @@ EUTelAlignGBL::EUTelAlignGBL(): Processor("EUTelAlignGBL") {
+  registerOptionalParameter("excludePlanes","Exclude planes from fit according to their sensor ids",_excludePlanes_sensorIDs ,std::vector<int>());
+  registerOptionalParameter("upstreamTriplet","The three sensors used as the upstream triplet", _upstream_triplet_ids, std::vector<int>{0,1,2});
+  registerOptionalParameter("downstreamTriplet","The three sensors used as the downstream triplet", _downstream_triplet_ids, std::vector<int>{3,4,5});
+  registerOptionalParameter("lastUpstreamSensor","The last plane (z-ordered) which still should be attached to the upstream triplet", _last_upstream_sensor, 2);
+  registerOptionalParameter("resolutionX","x-resolution of sensors (z-ordered) [mm]", _x_resolution_vec ,std::vector<float>());
+  registerOptionalParameter("resolutionY","y-resolution of sensors (z-ordered) [mm]", _y_resolution_vec ,std::vector<float>());
+  registerOptionalParameter("fixedPlanes","Fix sensor planes in the fit according to their sensor ids",_FixedPlanes_sensorIDs ,std::vector<int>());
+  registerOptionalParameter("maxTrackCandidatesTotal","Maximal number of track candidates (Total)",_maxTrackCandidatesTotal, 10000000);
+  registerOptionalParameter("maxTrackCandidates","Maximal number of track candidates",_maxTrackCandidates, 2000);
+  registerOptionalParameter("milleBinaryFilename","Name of the Millepede binary file",_binaryFilename, std::string{"mille.bin"});
+  registerOptionalParameter("alignMode","Number of alignment constants used. Available mode are:"
                               "\nXYZShifts - shifts in X and Y"
                               "\nXYShiftsRotZ - shifts in X and Y and rotation around the Z axis,"
                               "\nXYZShiftsRotZ - shifts in X,Y and Z and rotation around the Z axis",
-                              _alignModeString, std::string("XYShiftsRotZ"));
-  registerOptionalParameter("triCut", "Upstream triplet residual cut [um]", _triCut, 0.30);
-  registerOptionalParameter("driCut", "Downstream triplet residual cut [um]", _driCut, 0.40);
-  registerOptionalParameter("sixCut", "Upstream-Downstream Track matching cut [um]", _sixCut, 0.60);
-  registerOptionalParameter("slopeCut", "t(d)riplet slope cut [radian]", _slopeCut, 0.01);
-  registerOptionalParameter("GeneratePedeSteerfile","Generate a steering file for the pede program.",_generatePedeSteerfile, 0);
-  registerOptionalParameter("PedeSteerfileName","Name of the steering file for the pede program.",_pedeSteerfileName, string("steer_mille.txt"));
-  registerProcessorParameter("kappa","Global factor to Highland formula", _kappa, 1.0); // 1.0 means HL as is, 1.2 means 20% additional scattering
-
-
+                              _alignModeString, std::string{ "XYShiftsRotZ" });
+  registerOptionalParameter("upstreamTripletResidualCut", "Upstream triplet residual cut [mm]", _upTriResCut, 0.30);
+  registerOptionalParameter("downstreamTripletResidualCut", "Downstream triplet residual cut [mm]", _downTriResCut, 0.40);
```

Comment:
What about setting the same cuts in case the downstream one is not set, as @TCThenna suggested?
```
registerOptionalParameter("upstreamTripletResidualCut", "Upstream triplet residual cut [mm]", _upTriResCut, 0.30);
registerOptionalParameter("downstreamTripletResidualCut", "Downstream triplet residual cut [mm]", _downTriResCut, _upTriResCut);
```

The same for the slopes. I just tested it and it seems to work smoothly.
(edit: I actually don't know if this point belongs here, but it was raised in a parallel discussion)

Reply:
In my opinion, the problem with that is that if you generate a template it will show:

```
<!-- parameter key="upstreamTripletResidualCut" type=double value="0.30" -->
<!-- parameter key="downstreamTripletResidualCut" type=double value="0.30" -->
```

I.e.. suggesting that 0.30 is the default value if nothing else is set. This is not true however, not for the downstreamTripletResidualCut one at least.

Reply:
Ah, I see the problem. In any case, this was just a minor thing. Moreover, I just realized with the new nomenclature it is clearer that two cuts must be set and there is almost no room for mistake :)

Reply:
or we set no default at all for the downstreamSlope? 
The value to use heavily depends on the use case, so maybe no defaulted cut here, if this means less confusion ...

Reply:
What exactly do you suggest - set it to a mandatory parameter? I.e. `registerParameter(...)`